ui/components/DuplicatesDataTable.js

```diff
@@ -15,9 +15,9 @@ const DuplicatesDataTable = ({ view, rowData, classes }) => {
   const [, setSearchText] = useState(null);
   const [rowsPerPage, setRowsPerPage] = useState(10);
 
-  const kind = rowData[0].props?.children?.props?.children;
-  const model = rowData[3].props?.children?.props?.children;
-  const version = rowData[1].props?.children?.props?.children;
+  const kind = rowData[0]?.props?.children?.props?.children;
+  const model = rowData[3]?.props?.children?.props?.children;
+  const version = rowData[1]?.props?.children?.props?.children;
```

Comment:
@UsmanDev09 Please reference this by name, not index. This code is fragile.

@sudhanshutech will you take over this PR, please?